src/global/helpers/breakpoint.ts

```diff
@@ -0,0 +1,19 @@
+type Breakpoint = 'zero' | 'micro' | 'small' | 'medium' | 'wide' | 'large' | 'ultra';
```

Comment:
@kyubisation @mcilurzo 
should we consider screen resolution above ultra? (eg. my home monitor is 2580px wide)

Reply:
I improved the `isBreakpoint()` function so that it also covers larger screens, by allowing the possibility of setting only a `min-width`. 👍